11/jdk/zulu/Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM azul/zulu-openjdk-alpine:11.0.5
+FROM azul/zulu-openjdk-alpine:11.0.7
 MAINTAINER Forest Keepers <Jeroen.knoops@philips.com>
 
 RUN apk --no-cache add \
-      curl \
```

Comment:
Can you keep the original indenting? This gives some clear overview of what are arguments of `apk add` and the next command.